src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(string -> string.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
```suggestion
                .flatMap(n -> Arrays.stream(n.split(",")))
```
![image](https://user-images.githubusercontent.com/20326858/228491647-6f830e95-7a84-41d2-aba1-6fcdad75b21a.png)